pytorch_lightning/utilities/cli.py

```diff
@@ -293,10 +380,30 @@ def __init__(
             self.fit()
             self.after_fit()
 
+    @property
+    def registered_optimizers(self) -> Tuple[Type[Optimizer]]:
+        return tuple(OPTIMIZER_REGISTRY.values())
+
+    @property
+    def registered_lr_schedulers(self) -> Tuple[LRSchedulerType]:
+        return tuple(LR_SCHEDULER_REGISTRY.values())
+
     def init_parser(self, **kwargs: Any) -> LightningArgumentParser:
         """Method that instantiates the argument parser."""
         return LightningArgumentParser(**kwargs)
 
+    def sanetize_argv(self) -> None:
+        args = [idx for idx, v in enumerate(sys.argv) if v.startswith("--")]
```

Comment:
This does not take into consideration arguments as `-v True`

Reply:
In theory, this shouldn't be updated. Let me verify.